### ISSUE TITLE
Mouse sensitivty adjustments on button press not working.

### ISSUE THREAD

**Issue by Naughty-Maggot** [Ryochan7/DS4Windows#1733]
I use the Left thumb stick to control my mouse pointer on my "Windows" profile and have L2&R2 unbound but set as sensitivity adjusters. So holding L2 will reduce the cursor speed and holding R2 will increase the cursor speed. This no longer works. It used to, i can't recall the version number but it was a while ago.

Any help with this would be much appreciated as i also use this set-up for games where controllers can't be used. I find the ability to change the sensitivity with the triggers very helpful.

DS4 v.2 (CUH-ZCT2U)
Windows 10
Version 2.2.1

**Comment by Ryochan7** [comment]
For some reason, that modifier does not work on L2. It does work for R2 strangely enough. I'll look into it.

**Comment by Naughty-Maggot** [comment]
It doesn't work on either for me now. I also had it working on R2 the other day on one of my profiles but now i can't get either trigger to work with it.
In fact I can't get any modifiers to work in the extra section of the select action screen. Maybe the problem is located there? The rumble doesn't work and neither does the change colour tick box.

**Comment by Ryochan7** [comment]
Upon examining this further, the routine is working fine. I didn't take into consideration that the default profile dead zone for L2 and R2 is 0.00. That means the mouse sensitivity bound to R2 would be used at all times. You need to set a dead zone on L2 and R2 if you want to map a change in mouse sensitivity to either trigger. Not a bug.

**Comment by Naughty-Maggot** [comment]
That doesn't resolve my issue unfortunately. Upon further testing I can confirm that nothing in the "Extra" section works with L1,L2,L3,R1,R2,R3 or even the directional inputs of either thumb-stick. Every other button works with the Rumble, Color change and the Sensitivity adjustments in that menu. I've tried adjusting every option available to me and nothing I've tried has resolved this issue.
